.circleci/config.yml

```
@@ -62,11 +72,13 @@ jobs:
     <<: *container_config_node
     steps:
       - checkout
-      - node/install-npm:
+      - node/install-npm: null
```

Question: is this change expected?
```suggestion
      - node/install-npm: null
```

---

@lbarrous Please ask about this is expected or not to Platform team
https://financialtimes.slack.com/archives/C010BC61NDS/p1647438855529749?thread_ts=1647336271.361169&cid=C010BC61NDS

---

good spot - sorry this change isn't expected, we will take a look into this. once you remove this change in line 75 it is ready to merge

---

Thank you @serena97.
@lbarrous Could you remove this change please?